### PR TITLE
fix: Add anvil sizes to WDS Checkbox and WDS Table widgets

### LINKED ISSUE #28313: [Bug]: WDS checkbox and Table widget in Anvil have odd sizes
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Screenshot 2023-10-23 at 11 55 58 PM
Screenshot 2023-10-23 at 11 56 19 PM

### Steps To Reproduce

1. Go to https://ce-28238.dp.appsmith.com/
2. Add a checkbox and a table widget to an app

### Public Sample App

_No response_

### Environment

Deploy Preview

### Issue video log

_No response_

### Version

https://ce-28238.dp.appsmith.com/

### PR DESCRIPTION
**Pull request by riodeuno**
## Description
- Anvil needs a minHeight for widgets that occupy all of the parent's height.
- This way the bounding box assigned to the widget will be appropriate in height.

#### PR fixes following issue(s)
Fixes #28313

#### Type of change
- Bug fix (non-breaking change which fixes an issue)

#### How Has This Been Tested?
- [x] Manual
- [ ] JUnit
- [ ] Jest
- [ ] Cypress

### CODE REVIEW

File: app/client/src/widgets/wds/WDSTableWidget/widget/index.tsx [hunk 1]
```diff
@@ -373,6 +374,17 @@ export class WDSTableWidget extends BaseWidget<TableWidgetProps, WidgetState> {
     };
   }
 
+  static getAnvilConfig(): AnvilConfig | null {
```
Comment (KelvinOm) [inline app/client/src/widgets/wds/WDSTableWidget/widget/index.tsx:377]: Can we get rid of  `getConfig` `getDefaults` `getAutoLayoutConfig` methods and related configs (maybe something else). We have to remove unnecessary options and merge it all into one config for Anvil. The code related to configs needs to be simplified.

cc @dhruvikn @jsartisan

File: app/client/src/widgets/wds/WDSTableWidget/widget/index.tsx [hunk 2]
```diff
@@ -373,6 +374,17 @@ export class WDSTableWidget extends BaseWidget<TableWidgetProps, WidgetState> {
...
+    return {
+      widgetSize: {
+        maxHeight: {},
```
Comment (KelvinOm) [inline app/client/src/widgets/wds/WDSTableWidget/widget/index.tsx:380]: Are empty objects required here? Why?

**Comment by riodeuno** [comment]
Closing in favour of #28315 28315

**Pull request closed without merging**